ovve_ui/ovve_ui.py

```diff
@@ -28,26 +28,29 @@
                              initializeRespiratoryRateWidget,
                              initializeTidalVolumeWidget,
                              initializeIERatioWidget, initializeAlarmWidget,
-                             initializeGraphWidget)
+                             initializeGraphWidget, initializeSettingsWidget)
 from utils.params import Params
 from utils.settings import Settings
 from utils.alarms import Alarms
 from utils.comms_simulator import CommsSimulator
 from utils.comms_link import CommsLink
 from utils.ranges import Ranges
 
+
 class MainWindow(QWidget):
     new_settings_signal = pyqtSignal(dict)
 
-    def __init__(self, port: str, is_sim: bool=False) -> None:
+    def __init__(self, port: str, is_sim: bool = False) -> None:
         super().__init__()
         self.settings = Settings()
         self.local_settings = Settings()  # local settings are changed with UI
         self.params = Params()
         self.ranges = Ranges()
-        
-        
-        self.fullscreen = False
+
+        self.fullscreen = False  #TODO: Change Back
```

Comment:
You need to change this back, right?   You could also add a cmdline option to launch in windowed mode.